### ISSUE TITLE
 [Expo] ERROR  TypeError: Super expression must either be null or a function, js engine: hermes

### ISSUE THREAD

**Issue by geoffcfchen** [livekit/client-sdk-react-native#186]
**Describe the bug**

I am using the Expo managed workflow and EAS build. After following the installation instructions from (https://docs.livekit.io/home/quickstarts/expo/), I encountered an error. Specifically, when I import { registerGlobals } from @livekit/react-native, I receive the following error:

 ERROR  TypeError: Super expression must either be null or a function, js engine: hermes

**Expected behavior**

No error after following installation instructions (https://docs.livekit.io/home/quickstarts/expo/)

**Screenshots**

![IMG_7026](https://github.com/user-attachments/assets/de7b18c5-a740-48a0-a5ab-d16c6580649b)

**Device Info:**

 - Device: iPhone 16 
 - OS: 18.2

**Dependencies Info (please reference your package-lock.json or yarn.lock file):**

packapge.json
```
  "dependencies": {
    "expo": "~50.0.20",
    "@config-plugins/react-native-callkeep": "^7.0.0",
    "@config-plugins/react-native-pdf": "7.0.0",
    "@config-plugins/react-native-webrtc": "^8.0.0",
    "@livekit/react-native": "^2.5.0",
    "@livekit/react-native-expo-plugin": "^1.0.0",
    "@livekit/react-native-webrtc": "^125.0.6",
```
    
 app.json
 
     "plugins": [
      "@config-plugins/react-native-webrtc",
      "@config-plugins/react-native-callkeep",

**Additional context**

**Comment by geoffcfchen** [comment]
@davidliu  thanks for acting on this error. is there any work around for this issue or i can downgrade the version of livekit? Thanks again for the advice.

**Comment by davidliu** [comment]
Planning to push a fix in a day or two.

If you're not using E2EE, you should be able to just use the previous version of the SDK and `@livekit/react-native-webrtc`.

**Comment by geoffcfchen** [comment]
Thank you @davidliu  Just use a patch to fix it temporarily! Thank you so much for quickly fixing the problem and submitting the pull request (so I know how to fix it).

**Comment by davidliu** [comment]
@geoffcfchen can you confirm that it works for you? This didn't seem to error for me so I can't repro it.

**Comment by geoffcfchen** [comment]
@davidliu Yes. 

After modify this line from

import { Event, EventTarget, defineEventAttribute } from 'event-target-shim';

to 

import { Event, EventTarget, defineEventAttribute } from 'event-target-shim/index';

the error is gone.

**Comment by davidliu** [comment]
Fixed in 2.5.1.
